### ISSUE TITLE
Facet pop-ups can spawn under elements of website pages

### ISSUE THREAD

**Issue by ryanalcorn88** [mendezc1/GenderMagRecordersAssistant#38]
* **What Operating System?**
Windows

* **What steps will reproduce the issue?**
Start the tool up on a website that has an interactive toolbar at the top. Enter in information until the bullet list containing the five facets appears which is after picking the persona. Click one of the links for the five facets. A facet pop-up should appear in the top right.

* **What would you expect to be the outcome?**
I would expect the facet pop-up to spawn above all of the elements on the website page. If not the user can accidentally click on an element of the website page when trying to move or close the pop-up.

* **All these details will help people to fix any potential bugs?**
I tested the tool out on multiple websites. Some had more interactive elements and others didn't. It seemed to mainly happen underneath toolbars at the top of websites. I also tried it out without my ad-blocker on and in incognito mode. It spawned the pop-ups underneath certain elements in those scenarios also.

![example1](https://user-images.githubusercontent.com/39017022/58442725-f4672080-80a1-11e9-8f8e-aac19bb9daf9.jpg)

![example3](https://user-images.githubusercontent.com/39017022/58442731-00eb7900-80a2-11e9-9d25-560d86c18dfb.jpg)

**Comment by roseg31** [comment]
I'm unable to reproduce this issue - could you tell me which versions of windows and chrome you're using?

**Comment by ryanalcorn88** [comment]
I'm using Windows 10 and my Chrome version is Version 74.0.3729.169 (Official Build) (64-bit). I tried it again today and it appears to be working fine now. They are spawning above the toolbars. Also, the screenshot pop-up isn't going underneath certain elements of pages like in my second screenshot. I saw that you submitted a patch dealing with the facet pop-ups a day ago. Maybe that fixed it too.
